testing/integration/error_ext_stack/error_ext_stack.out

```diff
@@ -0,0 +1,4 @@
+[ERR] Error: Failed
+[ERR]     at innertTrowInExt (checkin:throwInExt:11:9)
+[ERR]     at throwInExt (checkin:throwInExt:7:3)
+[ERR]     at test:///integration/error_ext_stack/error_ext_stack.ts:3:1
```

Comment:
I believe this line number is wrong -- is this an existing bug?

Reply:
I don't know. `error_with_stack` presents the same issue, the line number is off by one.

Reply:
I suspect we have something wrong in the checkin transpiler. We can fix this one later.